Remotion/ObjectBinding/Sample/Person.cs

```diff
@@ -252,6 +252,10 @@ public string[] CV
       set { _cv = value; }
     }
 
+    [XmlIgnore]
+    [MultiLingualName("IstVolljährig", "")]
+    public bool IsOfLegalAge => (DateTime.Now - DateOfBirth).Days > 365 * 18;
```

Comment:
Just for fun: we actually need to use the Year property, I think. otherwise, there's a couple-of-days shift due to leap years.